src/components/DoubleCheckModal/DoubleCheckModal.tsx

```diff
@@ -1,16 +1,8 @@
 import React from "react";
 import { Box, Button, Modal, Typography } from "@mui/material";
+import "./doubleCheckModal.css";
+import "../../index.css";
```

Comment:
the index.css is already imported on the index.tsx file so I believe it doesn't need to be imported here

Reply:
fixed